client/src/index.css

```diff
@@ -0,0 +1,68 @@
+:root {
```

Comment:
We should rename it to `globals.scss` and remove no longer used properties, which would make our file like so:
```css
@use '../src/style/variables.scss' as *;
@use '../src/style/typography.module.scss' as *;

html,
body {
    font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
    line-height: 1.5;
    font-weight: 400;

    color-scheme: light dark;
    color: rgba(255, 255, 255, 0.87);
    background-color: #242424;

    font-synthesis: none;
    text-rendering: optimizeLegibility;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
}

h1 {
    @extend %h1;
}

h2 {
    @extend %h2;
}

h3 {
    @extend %h3;
}

h4 {
    @extend %h4;
}

h5 {
    @extend %h5;
}

p {
    @extend %p;
}
```